bitnami/thanos/values-production.yaml

```diff
@@ -236,6 +236,57 @@ querier:
     ##
     # maxUnavailable: 1
 
+  ## Configure the ingress resource that allows you to access Thanos Querier
```

Comment:
Can you remove this section as you did in the regular _values.yaml_?

Reply:
yes sorry about that. fixed now